OFEP-metric-hooks.md

```diff
@@ -0,0 +1,85 @@
+## OFEP-hook-for-metrics
+
+## State: PENDING REVIEW
```

Comment:
@Kavindu-Dodan I will update this to `APPROVED` if/when we get the required approvals.

Reply:
Thank you :) let's wait for more feedback/reviews